src/server/editorServices.ts

```diff
@@ -158,6 +158,11 @@ namespace ts.server {
         "References": {
             match: /^(.*\/_references\.js)$/i,
             exclude: [["^", 1, "$"]]
+        },
+        "Datatables.net": {
+            // e.g. /wwwroot/lib/datatables.all.min.js
+            match: /(jquery\.)?dataTables(\.all)?(\.min)?\.js$/i,
+            types: ["datatables.net"]

```

Comment:
No need to add this to the baked in default safelist also. It's not that common a library. Just having it in the typesMap.json should be fine.

Reply:
Removed.